frontend/src/charts/DisparityBarChart.tsx

```diff
@@ -33,6 +33,19 @@ function getSpec(
   // We use nested ternerys to determine the label's y axis delta based on the number of lines in the label to vertically align
   const AXIS_LABEL_Y_DELTA = `length(${MULTILINE_LABEL}) == 2 ? -3 : length(${MULTILINE_LABEL}) > 2 ? -7 : 5`;
 
+  function maxValueInField(field: string) {
+    return Math.max(
+      ...data
+        .map((row) => row[field])
+        .filter((value: number | undefined) => !!value)
```

Comment:
`!!value` will cause you to filter out `0` which is a valid number (though in practice this will probably never be an issue). Better to do `!== undefined`